csp_billing_adapter_local/plugin.py

```diff
@@ -44,6 +46,13 @@ def get_local_path(filename):
     return local_storage_path
 
 
+@csp_billing_adapter.hookimpl
+def setup_adapter(config: Config):
+    log_to_file = logging.FileHandler(CSP_LOG_FILEPATH)
+    csp_log = logging.getLogger(LOGGER_NAME)
```

Comment:
should either be `csp_log = log` or `log = logging.getLogger(LOGGER_NAME)` on line 37 should be removed. Why create the module variable at the top level when it is not being used?